02 - Memtables, Joins, and Selectors.ipynb

```diff
@@ -463,7 +463,7 @@
    "outputs": [],
```

Comment:
<ol><li>Is there a reason why we use a dictionary here? In the docs examples <a href="https://ibis-project.org/reference/expression-tables#ibis.expr.types.relations.Table.rename" target="_blank">here</a> we go over multiple ways of renaming columns, but there is no dictionary like example. Why not just doing `.r<code>.rename(new_name="old_name")</code></li><li>Maybe used the word "renamed" in the comments instead of relabeled just to be consistent.</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis-tutorial/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>